src/Compilers/Core/Portable/Binding/BindingDiagnosticBag.cs

```diff
@@ -283,6 +293,9 @@ internal bool Add(UseSiteInfo<TAssemblySymbol> info, Location location)
         }
 
         internal bool ReportUseSiteDiagnostic(DiagnosticInfo? info, Location location)
+            => ReportUseSiteDiagnostic(info, location, static location => location);
+
+        internal bool ReportUseSiteDiagnostic<TData>(DiagnosticInfo? info, TData data, Func<TData, Location> getLocation)
```

Comment:
>, TData data, Func<TData, Location> getLocation [](http://example.com/codeflow?start=73&length=47)

Similar feedback about the order of parameters

Reply:
done.